beets/mediafile.py

```diff
@@ -799,6 +799,9 @@ def fetch(self, mutagen_file):
             return []
 
     def store(self, mutagen_file, values):
+        if values == []:
```

Comment:
Perfect, I didn't know if you preferred that or preferred to compare with []. Probably after seeing some assertEqual with [] I thought you preferred to compare with []. I'll fix it